README.md

```diff
@@ -12,3 +12,53 @@ Grafana or the like.
 This is a [Haskell stack tools](https://docs.haskellstack.org/) based
 project.  Given a working `stack`, one should be able to execute
 `stack build` which will fetch all of the dependencies and build.
+
+After building, you will have several executables.
+
+teslauth: Authenticates to Tesla's API (with your credentials) and
+stores them in a local database.  _For obvious reasons_ treat this
+carefully.
+
+teslacatcher: Drains an MQTT topic to a sqlite database.
+
+tesladb: Runs continuiously polling Tesla's API to write to a database
+and MQTT, by default.
+
+tesladbfix: Filters out fields that have bad timestamps to a table
+named "rejects" in the database.
```

Comment:
I should probably remove this.  It was a migration tool, but I liked it and thought I might need to do another migration someday.  YAGNI?

Reply:
well, I'll document it since it's there, but we can remove it later I guess.